money-api/src/main/java/com/comcast/money/api/SpanInfo.java

```diff
@@ -26,32 +30,67 @@
      */
     Map<String, Note<?>> notes();
 
+    /**
+     * @return a list of all of the events that were recorded on the span.
+     */
+    List<Event> events();
+
     /**
      * @return the time in milliseconds when this span was started
      */
-    Long startTimeMillis();
+    long startTimeMillis();
```

Comment:
Changed these methods to return `long` as it avoids the allocation and it doesn't seem that these methods are ever intended to return `null`.